cypress/plugins/index.js

```diff
@@ -0,0 +1,14 @@
+const compareSnapshotsPlugin = require('cypress-visual-regression/dist/plugin');
+
+module.exports = (on, config) => {
+  compareSnapshotsPlugin(on, config);
+
+  require('cypress-mochawesome-reporter/plugin')(on);
+
+  // force color profile
+  on('before:browser:launch', (browser = {}, launchOptions) => {
```

Comment:
This was a try, to fix this color profile problem, but I found only a solution for tests with chrome and I used electron instead of chrome. Maybe we can discuss which browser we would like to use for these visual regression tests.

Reply:
I don't think color profile is really an issue but fonts are!